lib/ArduinoNative/src/Arduino.cpp

```diff
@@ -446,11 +483,17 @@ static int createConfigFile(const PrgArguments& prgArgs)
                 const size_t        JSON_DOC_SIZE = 2048U;
                 DynamicJsonDocument jsonDoc(JSON_DOC_SIZE);
 
-                jsonDoc["robotName"]    = prgArgs.robotName;
-                jsonDoc["WIFI"]["SSID"] = WIFI_SSID_DEFAULT;
-                jsonDoc["WIFI"]["PSWD"] = WIFI_PASSPHRASE_DEFAULT;
-                jsonDoc["MQTT"]["HOST"] = prgArgs.mqttHost;
-                jsonDoc["MQTT"]["PORT"] = prgArgs.mqttPort;
+                jsonDoc["robotName"]             = prgArgs.robotName;
+                jsonDoc["WIFI"]["SSID"]          = WIFI_SSID_DEFAULT;
+                jsonDoc["WIFI"]["PSWD"]          = WIFI_PASSPHRASE_DEFAULT;
+                jsonDoc["MQTT"]["HOST"]          = prgArgs.mqttHost;
+                jsonDoc["MQTT"]["PORT"]          = prgArgs.mqttPort;
+                jsonDoc["AP"]["SSID"]            = AP_SSID_DEFAULT;
+                jsonDoc["AP"]["PSWD"]            = AP_PASSPHRASE_DEFAULT;
+                jsonDoc["WEBSERVER"]["USER"]     = WEBSERVER_USER_DEFAULT;
+                jsonDoc["WEBSERVER"]["PSWD"]     = WEBSERVER_PASSPHRASE_DEFAULT;
+                jsonDoc["PLATOON"]["PLATOON_ID"] = prgArgs.platoonId;
+                jsonDoc["PLATOON"]["VEHICLE_ID"] = prgArgs.vehicleId;

```

Comment:
Can all these keys be made "static const char xx[] = "XX";

Reply:
What about something like this:
```
namespace configKeys
{
    /** Robot name */
    static const char ROBOT_NAME[] = "robotName";

    /** WiFi */
    static const char WIFI[] = "wifi";

    /** MQTT */
    static const char MQTT[] = "mqtt";

    /** Access point */
    static const char AP[] = "ap";

    /** WebServer */
    static const char WEBSERVER[] = "webServer";

    /** Platoon */
    static const char PLATOON[] = "platoon";

    /** SSID */
    static const char SSID[] = "ssid";

    /** Passphrase */
    static const char PASSWORD[] = "pswd";

    /** Host */
    static const char HOST[] = "host";

    /** Port */
    static const char PORT[] = "port";

    /** User */
    static const char USER[] = "user";

    /** Platoon ID */
    static const char PLATOON_ID[] = "platoonId";

    /** Vehicle ID */
    static const char VEHICLE_ID[] = "vehicleId";

} // namespace configKeys
```

Reply:
It could be extracted into a separate file to prevent code duplication in SettingsHandler.cpp but I am not sure where such file should be located